draft-ietf-scitt-architecture.md

```diff
@@ -192,6 +192,11 @@ Auditor:
 
 : an entity that checks the correctness and consistency of all Transparent Statements issued by a Transparency Service.
 
+Client:
+
+: a process, user or another service that externally connects to the Transparency Service.
```

Comment:
lets take a normative definition of client from another spec if we can... perhaps

https://datatracker.ietf.org/doc/html/rfc6749#section-1.1

```
client
      An application making protected resource requests on behalf of the
      resource owner and with its authorization.  The term "client" does
      not imply any particular implementation characteristics (e.g.,
      whether the application executes on a server, a desktop, or other
      devices).

```

Reply:
I like this suggestion Orie.

Reply:
the definition is OAuth specific ("protected resource" is an OAuth term) and needs to be adapted.

>An application making registration requests to a Transparency Service on behalf of a person holding a claim (who may or may not be the issuer).

Reply:
Re my previous recommendation around RP was based in the applicability of OAUTH terms and concepts, so in that vein, I support this definition of client for the time being.

Reply:
I've updated above text